src/components/InputText/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './InputText.css';
+
+const InputText = props => {
+    return <input 
```

Comment:
from line 5 - 9, 2 space indenting required thank you

Reply:
@Mubangizi this has been fixed.